code/go/0chain.net/smartcontract/dbs/event/process.go

```diff
@@ -45,6 +45,11 @@ func (edb *EventDb) AddEvents(ctx context.Context, events []Event) {
 		switch EventType(event.Type) {
 		case TypeStats:
 			err = edb.addStat(event)
+		case TypeError:
```

Comment:
Where is this being used?

Reply:
EmitError is called in the following lines
https://github.com/0chain/0chain/blob/staging/code/go/0chain.net/chaincore/chain/state.go#L202
https://github.com/0chain/0chain/blob/staging/code/go/0chain.net/chaincore/chain/state.go#L213

Which Eventually adds this to events array
https://github.com/0chain/0chain/blob/staging/code/go/0chain.net/chaincore/chain/state/state_context.go#L198

Reply:
Did we test changes on local chain post integration?

Reply:
![image](https://user-images.githubusercontent.com/11261548/150215465-0a8264ff-f8aa-4f6f-805c-e44b49c5fc39.png)
Yep ... 
But not sure how the errors happened.... 😅. Just started the chain to encounter these error

Will add gorm.Model
But as the ticket mentioned just two columns did not add it.

Reply:
![image](https://user-images.githubusercontent.com/11261548/150232045-62538a03-1e53-465d-b08d-bec8026272c3.png)
this is after adding gorm.Model.